src/components/options/main.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+import _ from "lodash";
+
+import {
+  StoreKey,
+  CBC_CONFIG,
+  NWI_CONFIG,
+  RSA_CONFIG,
+} from "../../js/shared/constants";
+import LocalStore from "../../js/shared/local_store";
+import { onMount } from "svelte";
+
+// Inject Svelte components into the page.
+import Footer from "./footer.svelte";
+import Header from "./header.svelte";
+import Providers from "./providers/main.svelte";
+import Settings from "./settings/main.svelte";
+import SpecialProvider from "./special/main.svelte";
+
+// Global variable for store initial settings (before user changes).
+let initData = {};
+
+let providersTab;
+
+function updateTabsVisibility(data) {
+  const current = _.get(data, "detail");
+  const pages = document.querySelectorAll("main section");
+  for (let i = 0; i < pages.length; i++) {
+    const pageAttr = pages[i].getAttribute("data-tab");
+    if (!_.isEmpty(pageAttr)) {
+      pages[i].style.display = pageAttr === current ? "block" : "none";
+    }
+  }
+}
```

Comment:
Now that everything is on Svelte, you don't need to use plain Javascript code to show/hide the tabs, you can use instead `{#if}...{/if}` blocks